MovieApp/MovieApp/NetworkClient/NetworkClientProtocol.swift

```diff
@@ -0,0 +1,5 @@
+protocol NetworkClientProtocol {
```

Comment:
This is called `NetworkClientProtocol` but it's specific to movie client because you've called the function `Movie`.